### ISSUE TITLE
Mypy 0.590 returns Invalid type for  "sqlite3.dbapi2.Binary"

### ISSUE THREAD

**Issue by jenshnielsen** [python/mypy#4916]
The following code 

```python
import sqlite3

def _adapt_array() -> sqlite3.Binary:
    return sqlite3.Binary()
```

passes cleanly with mypy 0.580 but fails with mypy 0.590
```
test.py:3: error: Invalid type "sqlite3.dbapi2.Binary"
```
With mypy 0.580 this does not produce any error.

Python 3.6.5 on macOS 10.13.4 no flags passed to mypy 

mypy gitmaster 3fe13a182d8a4ff48b6b768d4cc6e25d0f380e18 has the same issue. 

I don't know if this is an issue in mypy or typesheed perhaps related to https://github.com/python/typeshed/pull/2026

**Comment by JelleZijlstra** [comment]
Yes, this is due to the typeshed change you mentioned (so ideally this issue should be moved to typeshed). I changed `Binary` from `Any` to `Type[Any]`. I submitted python/typeshed#2057 to fix it.

**Comment by jenshnielsen** [comment]
@JelleZijlstra Thanks I can move the issue

**Comment by jenshnielsen** [comment]
replaced by https://github.com/python/typeshed/issues/2060
